### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by pinoelefante**

### CODE REVIEW

File: resources/language/resource.language.it_it/strings.po [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+msgid "There is an update available for [B]Widevine CDM[/B]."
+msgstr "E' disponibile un aggiornamento di [B]Widevine CDM[/B]."
+
+msgctxt "#30034"
+msgid "Update"
+msgstr "Aggiorna"
+
+msgctxt "#30035"
+msgid "[B]loop[/B] is still loaded"
+msgstr "[B]ciclo[/B] ancora in corso"
```
Comment (emilsvennesson) [inline resources/language/resource.language.it_it/strings.po]: `loop` is a kernel module and should not be translated.

**Comment by emilsvennesson** [comment]
Hi and thank you for the translation! :-)

I made one small remark but other than that I'll happily merge this. :-)

**Comment by emilsvennesson** [comment]
@pinoelefante Thank you!

**Comment by pinoelefante** [comment]
@emilsvennesson i didn't know :)
thank you to you for this addon

**Pull request merged**